analyzer/python/ikos/report.py

```diff
@@ -1098,6 +1101,86 @@ def format(self, report):
             ])
 
 
+class JUnitFormatter(Formatter):
+    ''' JUnit.xml formatter '''
+
+    def get_timing_result(self, report):
+        ''' get the timing results ifor the analysis from the database '''
+        db = report.db
+        elapsed = 0.0
+        if db:
+            results = db.load_timing_results(True, True)
+            for result in results:
+                print(result)
+                if result[0] == 'ikos-analyzer':
+                    elapsed = result[1]
```

Comment:
You could return directly here
```suggestion
                    return result[1]
```